src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -132,6 +133,25 @@ class IOUAmountPage extends React.Component {
         });
     }
 
+    /**
+     * Return length of Amount string without commas.
+     *
+     * @param {String} amount
+     * @returns {Number}
+     */
+    amountLength(amount) {
+        const trailingZeroes = amount.match(/^0+/);
+        const trailingZeroesLength = trailingZeroes ? _.first(trailingZeroes).length : 0;
+        const absAmount = parseFloat((amount * 100).toFixed(2)).toString();
+
+        /*
+        We return sum of trailing zeroes length and absAmount length( which has value of the all digits after the decimal).
+        In case of zeroes, we add the trailing zeroes length to 2, which
+        represent places reserved for digits after decimal.
+        */
+        return trailingZeroesLength + (absAmount === '0' ? 2 : absAmount.length);
```

Comment:
```suggestion
        Return the sum of leading zeroes length and absolute amount length(including fraction digits). 
        When the absolute amount is 0, add 2 to the leading zeroes length to represent fraction digits.
        */
        return trailingZeroesLength +  absAmount === '0' ? 2 : absAmount.length;
```

Reply:
I addressed the comments, but am doing away with parentheses since === and ?: has lower precedence than +, it will break the code to put w/o it either at beginning or end.

Reply:
Oh sorry, I didn't know It was left accidentally on the suggestion. I meant to request only comment update